### PR TITLE
Update es.yaml

### PR DESCRIPTION
**Pull request by VendettaCalls**

### CODE REVIEW

File: locales/es.yaml [hunk 1]
```diff
@@ -99,23 +99,23 @@ sfc:
...
     gridMode: Vista en cuadrícula
     tableMode: Vista detallada
-    filterMode: Modo de filtrado
+    filterMode: Modo filtrado
     myListMode: MI LISTA
     copyFilterLink: Copiar los filtros actuales como enlace
     noFilterWarn: Seleccione primero al menos un filtro.
     exportMyList: Exportar MI LISTA a un archivo
-    myListEmptyWarn: "¡MI LISTA está vacía!\nNo hay nada que exportar."
+    myListEmptyWarn: "¡MI LISTA está vacía!\No hay nada que exportar."
```
Comment (zetaraku) [inline locales/es.yaml:107]: `\n` is the newline character. Keep it as-is.

File: locales/es.yaml [hunk 2]
```diff
@@ -4,26 +4,26 @@ hello: ¡Hola Mundo!
...
   cancel: Cancelar
   close: Cerrar
-  search: Buscar en
-  preview: Vista Previa
+  search: Buscar
+  preview: Vista previa
   download: Descargar
   goBack: Atrás
-  all: TODAS
+  all: Todas
```
Comment (zetaraku) [inline locales/es.yaml:23]: This entry is used as the placeholder in the text field.
The English translation is uppercased, but you can keep it capitalized if it looks more natural in Spanish.

File: locales/es.yaml [hunk 3]
```diff
@@ -44,47 +44,47 @@ term:
   internalLevel: Nivel interno
   minInternalLevel: Nivel mínimo interno
   maxInternalLevel: Nivel máximo interno
-  noteDesigner: Notas del Diseñador
-  totalNotes: TOTAL
+  noteDesigner: Diseñador del mapa
+  totalNotes: Total
```
Comment (zetaraku) [inline locales/es.yaml:48]: This entry is used as the table header of "Total Note Count" in the Detail View.
The English translation is uppercased, but you can keep it capitalized if it looks more natural in Spanish.

File: locales/es.yaml [hunk 4]
```diff
@@ -44,47 +44,47 @@ term:
...
   region: Región
 
 # Descriptions & Tooltips
 description:
   pending: Pendiente
   loading: Cargando datos…
   newSong: ¡Nueva canción!
-  unlockNeeded: Hay que desbloquear esta canción
+  unlockNeeded: Hay que desbloquear esta canción.
```
Comment (zetaraku) [inline locales/es.yaml:56]: This entry is used as the tooltip of locked sheets in the Sheet Dialog.
The English translation is intended to be short and without a period, just like the below `unavailableInRegion` one.

File: locales/es.yaml [hunk 5]
```diff
@@ -44,47 +44,47 @@ term:
...
   unavailableInRegion: 'No disponible en {region}'
   copied: ¡Copiado al portapapeles!
-  invalidNoteCounts: Recuentos de NOTA no válidos
-  filterResultEmpty: No se encuentra ninguna hoja que cumpla el filtro.
-  myListEmpty: Haga clic con el botón derecho del ratón o mantenga pulsada una hoja para añadirla a MI LISTA y, a continuación, podrá ver aquí las hojas seleccionadas.
-  drawPoolEmpty: El grupo de dibujo actual está vacío.
+  invalidNoteCounts: Recuentos de nota no válidos
```
Comment (zetaraku) [inline locales/es.yaml:59]: "NOTE" is uppercased just like "TAP", "HOLD", "SLIDE", "TOUCH", and "BREAK" in maimai, to emphasize that this is a special rhythm game term.

**Review by zetaraku — CHANGES_REQUESTED** [review]

File: locales/es.yaml (hunk 1 above)
Comment (VendettaCalls) [reply locales/es.yaml:107]: Sorry, that was just a misdelete

**Review by zetaraku — APPROVED** [review]
LGTM!

**Pull request merged**